upt_macports/templates/python.Portfile

```diff
@@ -3,7 +3,34 @@
 {% block portgroup %}
 PortGroup           python 1.0
 {% endblock %}
+{% block nameversion %}
+name                {{ pkg._pkgname() }}
+version             {{ pkg.upt_pkg.version }}
+revision            0
+{% endblock %}
+
+{% block dist_info %}
+homepage            {{ pkg.upt_pkg.homepage }}
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+{% endblock %}
 
 {% block versions %}
-python.versions     27 36 37
+python.versions     37
+
+if {${name} ne ${subport}} {
+
+    depends_build-append \
+                    port:py${python.version}-setuptools
+    {% if pkg.upt_pkg.requirements.run|length > 0 %}
+    depends_lib-append \
+                    {% for i in pkg.upt_pkg.requirements.run %}
+                    port:py${python.version}-{{i.name}} {% if not loop.last %}\
```

Comment:
This results in a trailing whitespace in the last entry.

Reply:
updated